components/chef-ui-library/src/molecules/chef-status-filter-group/chef-status-filter-group.scss

```diff
@@ -104,6 +118,11 @@ chef-status-filter-group {
         background: #E7E9E9;
         box-shadow: 0px 0px 0px 1px var(--chef-dark-grey) inset;
       }
+
+      &.disconnected {
+        background: #FADAFA;
```

Comment:
This exists in automate-ui as "fuschia-light" https://github.com/chef/automate/blob/42fb0448535b9df5a8b1acc1b29000ab836875c6/components/automate-ui/src/styles/_colors.scss#L19 (which I guess is a misspelling and it should be "fuchsia" FWIW). Is it necessary to unify these things? How would I do that?